_oasis

```diff
@@ -56,7 +56,7 @@ Library xenopsd
     Xenops_server_plugin,
     Xenops_server_skeleton,
     Scheduler,
-    Path,
+    Xpath,
```

Comment:
The module contains paths for resources in dom0. The `Xc_path` and `Xl_path` have paths specific to those backends. I'd be happy with any of the alternative names you suggested.